### DISCUSSION TITLE
Use the design from PCManFM (GTK) in the new PCManFM (QT6)?

### DISCUSSION THREAD

**Discussion by GrabbenD** [lxqt/lxqt#2487] [Q&A]
I found PCManFM QT6 (`wip_qt6`) version to be a lot faster and snappier than the current stable release (`1.3.0`/QT5) and since GTK version seems to be rarely updated I figured it's best to stick with QT.

However, QT version looks really bad out of the box (unlike the GTK version which is themed).
How can I get the same design as GTK in the new QT edition?

![pcmanfm](https://github.com/lxqt/lxqt/assets/45666572/7abc4873-b8af-4b17-8676-d2c8c923cf6f)

**Comment by tsujan** [comment]
Oh, your screenshot shows that you don't have any (active) Qt6 plugin where you use PCManFM-Qt and other Qt6 apps.

If you have a basic installation of LXQt and have installed all Qt6 branches, it'll be enough to set the EV `XDG_CURRENT_DESKTOP` to `LXQt`; otherwise, you might want to install `qt6ct` and activate it. Then, your Qt6 apps will have icons and will be styled in any way you prefer; like this (under LabWC):

![styled](https://github.com/lxqt/lxqt/assets/981076/f9c66ac0-8537-4f67-9fec-54cb56d0c618)

EDIT: LXQt's Qt (Qt6) plugins are more advanced than qt6ct, but the latter will do the job too.

**Reply by GrabbenD** [reply]
Thanks for the screenshot and quick response, it looks great!

My bad, I forgot to mention that I'm using Sway..

I came across this theme https://github.com/PapirusDevelopmentTeam/arc-kde, I've never themed my Linux desktop before (and I've been using it for many, many years 😅 )

Is `qt6ct` approach the best alternative for Sway in that case?

**Reply by tsujan** [reply]
> Is qt6ct approach the best alternative for Sway in that case?

If you don't have a basic installation of LXQt, yes.

Please also note that some Qt styles may have not been ported to Qt6 yet, but your main problem is the absence of icons, not the widget style; `qt6ct` should fix that.
